### DISCUSSION TITLE
ERROR	PrismaClientValidationError: Unknown arg `customerId` in where.customerId for type UserWhereUniqueInput.

### DISCUSSION THREAD

**Discussion by LGmatrix13** [prisma/prisma#10891] [Q&A] [answered]
Hello,

I keep getting this error in nextjs: `ERROR	PrismaClientValidationError: Unknown arg `customerId` in where.customerId for type UserWhereUniqueInput.`

However, `customerId` exists in my schema and I have a the `@unique` directive as well.

Here is my prisma schema: 
```
model User {
  id            Int       @id @default(autoincrement())
  name          String?
  email         String?   @unique
  emailVerified DateTime? @map("email_verified")
  phoneNumber   String?
  customerId    String?   @unique
  image         String?
  description   String?
  address       String?
  createdAt     DateTime  @default(now()) @map("created_at")
  updatedAt     DateTime  @default(now()) @map("updated_at")
  subscribed    Boolean   @default(false)
  menu          Menu[]
  
  @@map("users")
}
```
 
And here is my code:
```
 await prisma.user.update({
          where: {
            customerId: event.data.object.customer,
          },
          data: {
            subscribed: true,
          },
        });
```

event.data.object.customer comes from a payload of my stripe webhook. It is a string and I have already tested that it returns correctly.

Any suggestions? Thank you!!!

**Comment by LGmatrix13** [comment, marked as answer]
Looks like I forgot to do a migration, my bad.

**Reply by ibraidab** [reply]
Hi im living the same situation, what you refer about a "migration"? 
The problem start when y pass to AWS server from localhost.
